default/stringtables/en.txt

```diff
@@ -339,11 +339,15 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''PP allocated to the transfer project are added to the imperial stockpile. The amount to transfer and how many times the transfer should repeat can be specified.
 
-Only transfers PP that are available at this planet; PP allocation on planets that are not supply-connected to here will not be affected by this project.
+Only PP that are produced in the same supply-group (and none from the imperial stockpile) are allocated to the stockpile transfer project.
 
-Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue.  This project is used to transfer PP to the stockpile even if there are things on the queue below it that could have been allocated available PP.'''
+Surplus PP (PP not allocated to anything on the queue) at this or any other location will be transferred to the stockpile with or without a stockpile transfer project on the queue. PP allocated to a stockpile transfer are not allocated to items lower in the production queue.
```

Comment:
Second sentence reads / sounds unrelated to the first. Needs to connect to it... so maybe `PP allocated to a stockpile transfer are sent to the stockpile, even if there are additional items on the queue to which the PP could have been allocated.`

Reply:
Order of build items matters, so there should be something like "below" "lower" "with less priority" "further down" "later"

Reply:
(and i think we are going in circles here, i will make a final suggestion and then leave it to you guys)